### PR TITLE
Removing accidental preview file

### PR DESCRIPTION
**Pull request by bergerhoffer**
Accidentally added a preview file in #251. Removing it now!

### CODE REVIEW

**Review by bredamc — APPROVED** [review]
LGTM :)

**Comment by bergerhoffer** [comment]
Thanks! Merging before waiting for 3 approvals since this is just maintenance misc.

**Pull request merged**
